### ISSUE TITLE
Export graph after creating visual

### ISSUE THREAD

**Issue by jsbasra** [microsoft/SandDance#571]
Create a button to export a snapshot as a image or pdf. I don't currently see any way to export visualization after it is made. Please add this functionality.

**Comment by danmarshall** [comment]
Hello, you can use this code:
```js
const sdcanvas = document.querySelector('.sanddance-gl canvas');
const base64PNGDataUrl = sdcanvas.toDataURL('image/png');
```

**Comment by danmarshall** [comment]
Related: #198

**Comment by jsbasra** [comment]
@danmarshall , thank you for your reply. However, I don't really see how to use that code inside the Azure Data Studio / VSCode extension. Would I somehow connect it to the webapp, and use this javascript code in a console?

**Comment by danmarshall** [comment]
Ok, it wasn't clear that you were in an ADS/VSCode context. Can I ask if you are using the Snapshots feature? If so, the snapshots created only use a thumbnail of the chart, and are meant to re-hydrate an interactive visualization. Are you able to take a screenshot with your device? Perhaps you can explain your scenario and detail the feature request.

**Comment by jsbasra** [comment]
Yes, I am able to take a snapshot, and I can see optoins to cut/copy/paste but it doesn't allow me to paste an image outside. I was hoping there was a button to export the snapshot as a pdf/image for adding to a report.  It could be that I am missing the option somewhere in the UI.

**Comment by danmarshall** [comment]
Related: #614
